libs/interpreter-lib/src/parsing-util.spec.ts

```diff
@@ -8,65 +8,47 @@ import { CachedLogger, DiagnosticSeverity } from '@jvalue/jayvee-execution';
 import {
   type JayveeServices,
   createJayveeServices,
+  initializeWorkspace,
 } from '@jvalue/jayvee-language-server';
 import {
-  type ParseHelperOptions,
   expectNoParserAndLexerErrors,
-  loadTestExtensions,
-  parseHelper,
-  readJvTestAssetHelper,
+  parseTestFileInWorkingDir,
 } from '@jvalue/jayvee-language-server/test';
-import { type AstNode, type LangiumDocument } from 'langium';
 import { NodeFileSystem } from 'langium/node';
 
 import { extractDocumentFromFile, validateDocument } from './parsing-util';
 
 describe('Validation of parsing-util', () => {
-  let parse: (
-    input: string,
-    options?: ParseHelperOptions,
-  ) => Promise<LangiumDocument<AstNode>>;
-
+  const WORKING_DIR = path.resolve(__dirname, '../test/assets/parsing-util/');
```

Comment:
Since we use esm now, shouldn't we restrain the use of `__dirname`?

Reply:
True! It seems to work, though, for the test. And it's used at some more places. I suggest to open a follow-up issue.

Reply:
#568